src/commands/package.js

```diff
@@ -59,10 +58,19 @@ export default asyncCommand({
 		let builddir = path.resolve(cwd, argv.builddir || 'build');
 		let dest = path.resolve(cwd, argv.dest || 'pkg', `${argv.name}.zip`);
 
+		// Get label and description from src/intl/en_US.json or from argument
+		const { label, description } = getZimletLabelDescription() || argv;
+
+		if (!(label && description)) {
+			return error(`label and description string are not provided as argument`);
+		}
+
+		createLocalizationPropertyFiles(argv.name);
```

Comment:
we should move creation of properties file logic to a single function, currently it's scattered